.github/workflows/actions.yml

```diff
@@ -0,0 +1,49 @@
+---
+name: actions for env
+on:
+  push:
+    branches: actions
+  pull_request:
+    branches: master
```

Comment:
I think branches wants a list:
```
 branches: [actions]
 branches: [master]
```

Reply:
I read this guide: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags

and one of example is the follow:
on:
  push:
    # Sequence of patterns matched against refs/heads
    branches:    
      # Push events on main branch
      - main
      # Push events to branches matching refs/heads/mona/octocat
      - 'mona/octocat'
      # Push events to branches matching refs/heads/releases/10
      - 'releases/**'

with one line for different branch or ref. 
But it seem that also the [] are allowed, for example:
 
Example using a list of events
# Triggers the workflow on push or pull request events
on: [push, pull_request]

Reply:
The example you quote says that
```
branches:
  -main
```
is correct, NOT `branches: main`